pom.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-  ~ Copyright © 2016 Cask Data, Inc.
+  ~ Copyright © 2016 CDAP
```

Comment:
are we doing this everywhere? Also, perhaps change the year?

Reply:
The script I am using (from @anew) does this.

Reply:
year?